src/validations.ts

```diff
@@ -12,8 +13,11 @@ export function validateEmail(
 
   if (!emailRegex.test(value)) {
     return commonErrorKey;
+  } else {
+    if(accentRegex.test(value)) {
+      return 'validation_messages.error_has_accents';
+    }
```

Comment:
Now, you can reduce the nesting  😀

```suggestion
  } else if(accentRegex.test(value)) {
    return 'validation_messages.error_has_accents';
  }
```

Reply:
fixed